css/values.css

```diff
@@ -0,0 +1,336 @@
+/*
+===================================
+===================================
+Our Values
+===================================
+===================================
+*/
+
+/* 
+===================================
+Section Styles (background, padding, etc.)
+===================================
+*/
+
+/* 
+===================================
+Hero
+===================================
+*/
+.hero {
+	background: linear-gradient(
+			260deg,
+			rgba(9, 9, 9, 0.8) 46.9%,
+			rgba(9, 9, 9, 0) 86.82%
+		),
+		url("../assets/images/pages/values/hero-values.png") top/cover;
+	display: flex;
+	align-items: center;
+}
+/* values heading layout */
+.hero .wrapper {
+	display: flex;
+	justify-content: flex-end;
+}
+/* values heading sizing */
+.hero-text-container {
+	width: 50%;
+	display: flex;
+	flex-direction: column;
+	gap: var(--spacing-sm);
+}
+
+/* 
+===================================
+Values
+===================================
+*/
+
+/* layout */
+.values-list,
+.values-row,
+.values-img-text {
+	display: flex;
+	gap: var(--spacing-sm);
+}
+
+.values-list,
+.values-img-text {
+	flex-direction: column;
+}
+
+/* card widths */
+.sustainability,
+.authenticity {
+	width: 60%;
+}
+
+.humanity,
+.multigenerational {
+	width: 40%;
+}
+
+.education,
+.innovation {
+	width: 50%;
+}
+
+/* general card styling */
+.values-card {
+	padding: 60px;
+	color: var(--text-color-light);
+	border-radius: var(--border-radius);
+
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+/* text layout*/
+.values-card .text {
+	display: flex;
+	flex-direction: column;
+	gap: var(--spacing-sm);
+}
+
+/* card specific */
+.humanity {
+	background-color: var(--color-copper);
+}
+
+.sustainability {
+	background: linear-gradient(
+			269deg,
+			#0e0e0e 46.26%,
```

Comment:
Can we replace the hexadecimal values with variables?

Reply:
I will add value page gradients in addition to hero section gradients in `global.css`

Reply:
Can these values be replaced now?